### DISCUSSION TITLE
Play 1.7 WS problem (header too large)

### DISCUSSION THREAD

**Discussion by qabi** [playframework/playframework#11564] [Q&A] [answered]
It is unclear if this is the correct place for asking Play! 1.7 questions?

I have a problem with using the WS module, where the server responds with a (Set-Cookie) header that is over 8152 bytes. This does not seem to be configurable? I checked the source of Netty and it looks very difficult to change this max-header-size value, through the WS class.

Any ideas?

**Comment by mkurz** [comment, marked as answer]
The Play 1 repo can be found here: https://github.com/playframework/play1
You should open an issue there or use another channel (not sure if they have mailing list or something).
